modules/icefinder.nf

```diff
@@ -18,24 +18,24 @@ process ICEFINDER {
 
     containerOptions {
         args = [
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/input.list:/install/ICEfinder_linux/input.list",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/gbk/:/install/ICEfinder_linux/gbk/",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/tmp/:/install/ICEfinder_linux/tmp/",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/result/:/install/ICEfinder_linux/result/",
+            "--bind ${input_list}:/install/ICEfinder_linux/input.list",
+            "--bind ${gbk_folder}:/install/ICEfinder_linux/gbk/",
+            "--bind ${tmp_folder}:/install/ICEfinder_linux/tmp/",
+            "--bind ${res_folder}:/install/ICEfinder_linux/result/",
             "--pwd /install/ICEfinder_linux/"
```

Comment:
Would it be worth making this usable with Docker too? like this (from the interproscan nf-module):
```
    containerOptions {
        if (workflow.containerEngine == 'singularity') {
            return "--bind ${interproscan_db}/data:/opt/interproscan-5.62-94.0/data"
        } else {
            return "-v ${interproscan_db}/data:/opt/interproscan-5.62-94.0/data"
        }
    }
```

Reply:
We have to do that. But, ATM the pipeline lacks a bit as that container needs to be built by the user (licensing issues). We will re-work that bit in due time